lib/migration.js

```diff
@@ -186,7 +186,7 @@ function mixinMigration(MySQL, mysql) {
     function actualize(propName, oldSettings) {
       var newSettings = m.properties[propName];
       if (newSettings && changed(newSettings, oldSettings)) {
-        var pName = self.client.escapeId(propName);
+        var pName = self.columnEscaped(model, propName);
```

Comment:
Right. But what I am not understanding is, if we specify the `columnName` explicitly to be something different than the property name, how would this work?